packages/zowe-explorer/src/configuration/Constants.ts

```diff
@@ -0,0 +1,344 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import * as path from "path";
+import * as vscode from "vscode";
+import { FileManagement, imperative, IZoweTreeNode, PersistenceSchemaEnum } from "@zowe/zowe-explorer-api";
+import { ZoweLogger } from "../tools";
+import { Profiles } from "../configuration";
+
+export namespace Constants {
+    export let SETTINGS_TEMP_FOLDER_LOCATION;
+    export let ZOWETEMPFOLDER: string;
+    export let ZOWE_TMP_FOLDER: string;
+    export let USS_DIR: string;
+    export let DS_DIR: string;
+    export let CONFIG_PATH; // set during activate
+    export let LOG: imperative.Logger;
```

Comment:
Thanks for moving `globals.ts` to `Constants.ts`, I like this change 👍 

If we intend to use this file strictly for constants, then I think we should only define simple types in the `Constants` class like strings and numbers, and avoid storing references to classes and defining functions in this file if possible.

Curious if the `LOG` property could be moved to a static property of the `ZoweLogger` class? Or do we need to keep it here for now to avoid circular imports?

Reply:
hmm what benefit would there be from moving the `LOG` property?